### DISCUSSION TITLE
请求更新下 死亡细胞v2.2

### DISCUSSION THREAD

**Discussion by CJM9752** [Gamuxorg/bbs#268] [投递和请求游戏]
游戏名称：死亡细胞
游戏名称(英文)：Dead Cells
下载地址 :（如果有的话）：https://down.ali213.net/pcgame/deadcells.html  /  Steam

**Comment by shouhuanxiaoji** [comment]
如没有特殊情况，例如当前版本有重大bug，一般不再更新，可以考虑购买正版，或者等待时机白嫖。
